src/types/ChampionSkin.ts

```diff
@@ -12,6 +12,13 @@ export interface ChampionSkinData {
  * A representation of a champion's skin data returned by meraki analytics.
  * These are used in the LoL wiki.
  */
+
+export interface MerakiSkinChroma {
+  name: string;
+  id: number;
+  chromaPath: string;
+}
```

Comment:
I've actually ended up splitting in into separate interface since prettier does some black magic here

![image](https://user-images.githubusercontent.com/23432278/185791335-2ce28430-d202-4bb9-b22c-17adaa9217bc.png)

Reply:
This is the ideal way to write it but a separate interface works